src/bin/test_window.c

```diff
@@ -13,12 +13,19 @@ int main() {
     struct graphics_color bg = {0,0,0};
     struct event e;
     int x = 0;
+    int count = 0;
     while (1) {
         int res = get_event(&e);
         if (res == 0) {
+            if (count > 10) {
+                clear();
+                count = 0;
+                x = 0;
+            }
             if (e.type == EVENT_KEYBOARD_PRESS) {
                 draw_char(x, 0, e.character, &c, &bg);
```

Comment:
Correct, this is an issue of whether the window border is a part of the window (in which case it makes sense to be able to draw on top of them) or outside the window (in which case, does the width of border count against the size of the window?). I would rather get the clear function in right now and tackle these larger issues later.